tests/integration/alerts_test.js

```diff
@@ -77,4 +77,23 @@ describe('Integration: Alerts Endpoints', function(){
         .end(done);
     });
   });
+
+  describe('PUT /v1/alerts/:id', function(){
+    afterEach(function(){
+      clientStub.call.restore();
+    });
+
+    it('return a model', function(done){
+      sinon.stub(clientStub, 'call')
+        .withArgs('overwritedefault', sinon.match({ id: '1' }), sinon.match.any)
```

Comment:
the default verb for a put on this url would be update. I overwrote the default verb with a new verb called 'overwriteDefault' using a non standard action in this service's metadata